components/discord_webhook/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@pipedream/discord_webhook",
+  "version": "1.0.0",
+  "description": "Pipedream Discord_Webook Components",
+  "main": "discord_webhook.app.js",
+  "keywords": [
+    "pipedream",
+    "discord_webhook"
+  ],
+  "homepage": "https://pipedream.com/apps/discord_webhook",
+  "author": "Pipedream <support@pipedream.com> (https://pipedream.com/)",
+  "license": "MIT",
+  "dependencies": {
+    "axios": "^0.21.1",
+    "form-data": "^4.0.0",
+    "request": "^2.88.2",
+    "fs": "^0.0.1-security"
```

Comment:
Did you add this manually or did this come from somewhere else? I don't believe we'll need it since `fs` is built-in.

Reply:
Updated to remove `fs` and `request`